network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/wide_table_block.html

```diff
@@ -64,4 +54,4 @@
             </tbody>
         </table>
     </div>
-{% endblock %}
+{% endblock block_content %}
```

Comment:
**Kudos** for updating this to comply with Djlint rules!

Since you've renamed and updated it, we can go ahead and remove [this line](https://github.com/MozillaFoundation/foundation.mozilla.org/blob/TP1-144-10332-add-wide-table-block/pyproject.toml#L126) from `pyproject.toml`.

For a bit of background, when we first introduced Djlint to the repo, we encountered a lot of errors from older files that didn’t adhere to the linting rules. Instead of fixing everything in one go,  we created this list of files and errors to ignore temporarily.

Reply:
Definitely did not know about this. Thanks for pointing it out @danielfmiranda!